airbyte-integrations/base-java/src/main/java/io/airbyte/integrations/javabase/JavaBaseConstants.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.javabase;
+
+import java.nio.file.Path;
+
+public class JavaBaseConstants {
+
+  public static String ENV_DESTINATION_CLASS = "DESTINATION_CLASS";
+  public static String ENV_DESTINATION_JAR_PATH = "DESTINATION_JAR_PATH";
+
+  public static String ARGS_CONFIG_KEY = "config";
+  public static String ARGS_SCHEMA_KEY = "schema";
+  public static String ARGS_STATE_KEY = "state";
+
+  public static String ARGS_CONFIG_DESC = "path to the json configuration file";
+  public static String ARGS_SCHEMA_DESC = "input path for the schema";
+  public static String ARGS_PATH_DESC = "path to the json-encoded state file";
+
+  public static Path LOCAL_MOUNT = Path.of("/local");
```

Comment:
This feels a bit weird since this doesn't necessarily match what is retrieved by `EnvConfigs`.

Reply:
that's a good point. i guess the local mount env variable should be set by the worker and then this program will read that env variable? does that work for you?

Reply:
i'm going to not try to figure this out until i get the csv destination PR done, but i will fix it in the next PR.